internal/kibana/data_view/schema.go

```diff
@@ -658,9 +673,13 @@ type apiFieldFormat struct {
 }
 
 type tfFieldFormatParamsV0 struct {
-	Pattern types.String `tfsdk:"pattern"`
+	Pattern       types.String `tfsdk:"pattern"`
+	Urltemplate   types.String `tfsdk:"urltemplate"`
+	Labeltemplate types.String `tfsdk:"labeltemplate"`
 }
 
 type apiFieldFormatParams struct {
-	Pattern string `tfsdk:"pattern" json:"pattern"`
+	Pattern       string `tfsdk:"pattern" json:"pattern,omitempty"`
+	Urltemplate   string `tfsdk:"urltemplate" json:"urlTemplate,omitempty"`
+	Labeltemplate string `tfsdk:"labeltemplate" json:"labelTemplate,omitempty"`
```

Comment:
These should potentially be `*string` since they're optional in the schema.

Reply:
That's exactly what I was missing! Thanks for the assist. Changes made and pushed.